tests/target-browser/target_browser_test.go

```diff
@@ -405,10 +407,54 @@ var _ = Describe("Target Browser Tests", func() {
 				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagOrderBy)))
 			})
 
+			It(fmt.Sprintf("Should sort in ascending order using flag %s with default value", flagOrderBy), func() {
+				args := []string{cmdGet, cmdBackupPlan}
+				backupPlanData := runCmdBackupPlan(args)
+				for index := 0; index < len(backupPlanData)-1; index++ {
+					Expect(backupPlanData[index].Name <= backupPlanData[index+1].Name).Should(BeTrue())
+				}
+			})
+
+			It(fmt.Sprintf("Should sort in ascending order name using default flag %s with zero value", flagOrderBy), func() {
+				args := []string{cmdGet, cmdBackupPlan, flagOrderBy, ""}
+				backupPlanData := runCmdBackupPlan(args)
+				for index := 0; index < len(backupPlanData)-1; index++ {
+					Expect(backupPlanData[index].Name <= backupPlanData[index+1].Name).Should(BeTrue())
+				}
+			})
+
+			It(fmt.Sprintf("Should sort in ascending order name using default flag %s with 'invalid' value", flagOrderBy), func() {
+				args := []string{cmdGet, cmdBackupPlan, flagOrderBy, "invalid"}
```

Comment:
server should raise error for invalid values.. update this to @niranjan-gavade or @vinod-trilio